### PR TITLE
Add HTTP "head" method support in Path model

### PR DESCRIPTION
**Pull request by emailchaos**

### CODE REVIEW

**Comment by webron** [comment]
Thanks for the submission. Two notes though:
1. Code changes need to be made against the develop_2.0 branch, so please resubmit it against it.
2. In terms of `@JsonPropertyOrder` I would push it at the end, or at least after `options`.

**Comment by bofalot** [comment]
I am interested in this change, thanks for fixing. Just my $0.02, but I would keep ```head``` after ```get``` in the ```@JsonPropertyOrder``` as that is the order listed at http://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html.

File: modules/swagger-models/src/main/java/io/swagger/models/Path.java [hunk 1]
```diff
@@ -49,6 +53,11 @@ public Path get(Operation get) {
         return this;
     }
 
+    public Path head(Operation get) {
```
Comment (bofalot) [inline modules/swagger-models/src/main/java/io/swagger/models/Path.java]: The function argument should be ```head``` not ```get```. This typo prevents this change from actually working.

**Comment by webron** [comment]
I'm afraid I have to close it as it's submitted against the wrong branch. Feel free to resubmit against develop_2.0.

**Comment by bofalot** [comment]
I've included this into my pull request https://github.com/swagger-api/swagger-core/pull/1217

**Pull request closed without merging**